From 153a786f76bd43be4a72c78c799e4d86efb6ac2c Mon Sep 17 00:00:00 2001
From: Christophe Kerello <christophe.kerello@foss.st.com>
Date: Fri, 20 Dec 2024 14:27:31 +0100
Subject: [PATCH 767/954] fix(stm32mp2-fdts): set SDMMC max frequency to 166
 MHz on stm32mp25

Set SDMMC max frequency to 166 MHz on stm32mp25.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/arm-trusted-firmware.git eb80ebe7c2d8cafed5af90bd59f2815130c98828 fix(stm32mp2-fdts): set SDMMC max frequency to 166 MHz on stm32mp25]
Change-Id: Ibc1eadcf7d942c9723bfe41d711a78371dfed99f
Signed-off-by: Christophe Kerello <christophe.kerello@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/tf-a/+/426152
Tested-by: Christophe KERELLO <christophe.kerello@st.com>
Reviewed-by: Maxime MERE <maxime.mere@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Reviewed-by: Christophe KERELLO <christophe.kerello@st.com>
Domain-Review: Christophe KERELLO <christophe.kerello@st.com>
---
 fdts/stm32mp251.dtsi | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/fdts/stm32mp251.dtsi b/fdts/stm32mp251.dtsi
index 378f11f960..dcf1a9963b 100644
--- a/fdts/stm32mp251.dtsi
+++ b/fdts/stm32mp251.dtsi
@@ -326,7 +326,7 @@
 				resets = <&rcc SDMMC1_R>;
 				cap-sd-highspeed;
 				cap-mmc-highspeed;
-				max-frequency = <120000000>;
+				max-frequency = <166000000>;
 				status = "disabled";
 			};
 
@@ -339,7 +339,7 @@
 				resets = <&rcc SDMMC2_R>;
 				cap-sd-highspeed;
 				cap-mmc-highspeed;
-				max-frequency = <120000000>;
+				max-frequency = <166000000>;
 				status = "disabled";
 			};
 		};
-- 
2.53.0

